indico/modules/events/registration/controllers/management/tickets.py

```diff
@@ -73,6 +74,11 @@ def _process(self):
                 'scope': 'registrants',
             }
         }
+        custom_qr_code_handlers = get_custom_ticket_qr_code_handlers()
+        if custom_qr_code_handlers:
+            qr_code_handler = custom_qr_code_handlers[0]
```

Comment:
I'd rather the return values from all handlers (ie a return a list of regexps). It's not common, but should be supported since there's not really a reason not to. And I could imagine some usecases where it may be useful...